testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -443,6 +443,17 @@ def test_pickle_reader(self, reader):
         assert_equal(len(reader), len(reader_p))
         assert_equal(reader.ts, reader_p.ts,
                      "Timestep is changed after pickling")
+    
+    def test_frame_collect_all_same(self, reader):
```

Comment:
Does this check that we are _not_ providing a copy of the ts?

I like the test. We just need to make an exception (or xfail) for DCDReader if we keep the current behavior until 3.0.

Reply:
Yep it checks that ts.positions is always the same.